### DISCUSSION TITLE
Disappearing vulnerabilities from the "Audit Vulnerabilities" tab

### DISCUSSION THREAD

**Discussion by redduxi** [DependencyTrack/dependency-track#3229] [Q&A]
Hello everybody,

I'm having a problem displaying vulnerabilities in the "Audit Vulnerabilities" tab. The fact is that when adding a dependency to the project, the list of vulnerabilities in the "Audit Vulnerabilities" tab begins to expand. At first everything is fine, but when the total number of vulnerabilities in the list crosses the mark of about 2500, they disappear. When reloading the page, "No matching records found" is displayed.
<img width="1400" alt="Screenshot 20 04 04" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/da622fb5-2e73-4d21-a89e-b9511dc7f0c4">
<img width="1385" alt="Screenshot 20 08 49" src="https://github.com/DependencyTrack/dependency-track/assets/75900543/eb2bacf6-b5dc-471d-a716-4c6fe00733bc">

We tried to increase the resources, but the problem remained.
The service was launched via helm-charts. The database used is aurora postgresql (version 15.4).

Maybe someone has encountered this or knows what the problem could be?

Thanks

**Comment by shardulbhisikar** [comment]
@redduxi - Do we have any updates on this? I am also facing the same issue of rendering the vulnerability data when the count is high.
@stevespringett - Please look into this and give solution. Thanks.

**Comment by quimicefa** [comment]
I'm facing the (I guess) same problem ... and in our scenario there are 2-3 vulnerabilities with a description including code examples that brokes the JSON parser. 

TL;DR -> edit the description for those vulnerabilites

https://github.com/DependencyTrack/dependency-track/issues/6136

**Comment by nscuro** [comment]
I think the particular issue in this discussion is caused by the findings API backing the "Audit Vulnerabilities" view not being paginated. Likely the request is timing out before all data could be loaded from the DB, transformed into JSON, and submitted to your browser.

This is addressed in v5 where we added proper pagination, so it only loads as many findings as fit on your screen (or as per your "Rows per page" setting).
